plugins/node/opentelemetry-instrumentation-knex/src/instrumentation.ts

```diff
@@ -122,6 +149,10 @@ export class KnexInstrumentation extends InstrumentationBase<KnexInstrumentation
 
   private createQueryWrapper(moduleVersion?: string) {
     const instrumentation = this;
+
+    // We need to bind it here, because `this` is not the same in the wrapper
+    const semConv = this._semconvStability;
```

Comment:
Or could use `instrumentation._semconvStability` below. This is fine though.

Reply:
doh, seems kind of obvious in hindsight ;)